src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlock.iOSmacOS.cs

```diff
@@ -21,5 +21,10 @@ private CGRect GetDrawRect(CGRect rect)
 
 			return rect;
 		}
+
+		public TextBlock()
+		{
+			_hyperlinks.CollectionChanged += HyperlinksOnCollectionChanged;
```

Comment:
Why does this need to be in every target? Is there something specific to a target that makes it not available to others?

Reply:
HyperlinksOnCollectionChanged just checks whether we should be subscribing to pointer events (previously was inside `UpdateHyperlinks`). I've basically included it on all targets except WASM as it wasn't included there in the first place (WASM has its own `UpdateHyperlinks`). Since the constructor for each target is not shared, I've had to manually add it to each target.